components/server/ee/src/prebuilds/gitlab-app.ts

```diff
@@ -124,6 +123,16 @@ export class GitLabApp {
         }
     }
 
+    private async getCommitInfo(user: User, repoURL: string, commitSHA: string) {
```

Comment:
[Same](https://github.com/gitpod-io/gitpod/pull/7785#discussion_r814631171) here.

Reply:
see my reply on the original comment